vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -6,13 +6,15 @@
         This script is used by packer to produce Windows AKS images.
 #>
 
-param()
+param (
+    $containerRuntime
+)
 
 $ErrorActionPreference = "Stop"
 
 filter Timestamp { "$(Get-Date -Format o): $_" }
 
-$global:containerdPackageUrl = "https://marosset.blob.core.windows.net/pub/containerd/containerd-0.0.87-public.zip"
+$global:containerdPackageUrl = "https://kubernetesartifacts.azureedge.net/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip"
```

Comment:
Please use `https://acs-mirror.azureedge.net/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip`

Reply:
@andyzhangx `https://mirror.azk8s.cn/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip` does not work. Could you help to take a look?

Reply:
Not for this PR: I think that we should add a test script to validate the URLs for mooncake later.

Reply:
pls use `https://mirror.azk8s.cn/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip` for mooncake

Reply:
@andyzhangx did you fix it? Now the link works.

Reply:
yes, just added a new proxy-forward:
```
  location /containerd/ {
    proxy_pass https://kubernetesartifacts.blob.core.chinacloudapi.cn/containerd/;
  }
```